src/modules/edc-demo/components/asset-viewer/asset-viewer.component.html

```diff
@@ -30,17 +30,17 @@
           <mat-list-item>
             <mat-icon mat-list-icon>info</mat-icon>
             <div class="asset-property" mat-line>Name</div>
-            <div mat-line>{{asset.name}}</div>
+            <div mat-line>{{asset.properties.mandatoryValue('edc','name')}}</div>
```

Comment:
I'm not sure this props should be considered "mandatory", as the asset properties are a free form field. `optionalValue` should be used instead. Same for the others

Reply:
Changed the function to optionalValue, you are right.